pipenv/cli.py

```diff
@@ -1020,12 +1022,12 @@ def check(three=None, python=False):
     # Assert each specified requirement.
     for marker, specifier in p.data['_meta']['requires'].items():
 
-            if marker in results:
-                try:
-                    assert results[marker] == specifier
-                except AssertionError:
-                    failed = True
-                    click.echo('Specifier {0} does not match {1} ({2}).'.format(crayons.green(marker), crayons.blue(specifier), crayons.red(results[marker])))
+        if marker in results:
```

Comment:
Sure thing, I'll keep it in mind for future PRs. I've noticed that maintainers are often split on the issue of whether to cleanup in a PR or on-the-go.